packages/vertexai/src/models/generative-model.test.ts

```diff
@@ -165,6 +165,35 @@ describe('GenerativeModel', () => {
     );
     restore();
   });
+  it('passes base model params through to ChatSession when there are no startChatParams', async () => {
```

Comment:
Can we add this case to the test below starting in line 197? That tests for a variety of params being passed not only to ChatSession but all the way down to sendMessage, which covers the whole path. And add the override case to 261? I got burned in the past by something getting passed to ChatSession but not to sendMessage.

Reply:
thanks for pointing that out- done